src/connection/BaseConnectionOptions.ts

```diff
@@ -110,6 +110,11 @@ export interface BaseConnectionOptions {
      */
     readonly extra?: any;
 
+    /**
+     * The directory where ormconfig has been read from
+     */
+    baseDirectory?: string;
```

Comment:
What happens when there's no ormconfig?

Why does this feature operate differently from how the standard database path resolution works for sqlite?